src/types/pagopa.ts

```diff
@@ -380,24 +377,10 @@ type Pay = t.TypeOf<typeof Pay>;
 export const PayRequest = repP(PayRequestPagoPA, 'data', Pay, 'PayRequest');
 
 export type PayRequest = t.TypeOf<typeof PayRequest>;
-
+// NEEDED
```

Comment:
what's the meaning of 'needed' in this context? (everything else is optional?)

Reply:
As you may have seen I tend to overcomment the code :-) but in this case is just a typo. I'll fix it